<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_a">
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>1221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_cp">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>27</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Change Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_logout">
       <property name="geometry">
        <rect>
         <x>1080</x>
         <y>27</y>
         <width>80</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_js">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>26</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>JOB SEEKER</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_jhlogo">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>111</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_jhlogo_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>111</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_sj">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>361</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_sj">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>211</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>Search Job</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_sj">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>180</y>
         <width>331</width>
         <height>301</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_aj">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>120</y>
        <width>361</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_aj">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>221</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>View Applied Job</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_aj">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>181</y>
         <width>331</width>
         <height>301</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_vp">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>120</y>
        <width>361</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_vp">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>231</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>View Profile</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_vp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>341</width>
         <height>301</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
